lib/Doctrine/ORM/Tools/SchemaTool.php

```diff
@@ -857,12 +856,9 @@ public function dropDatabase()
      */
     public function getDropDatabaseSQL()
     {
-        $schema = $this->schemaManager->createSchema();
-
-        $visitor = new DropSchemaSqlCollector($this->platform);
```

Comment:
`DropSchemaSqlCollector` will be deprecated in favor of another class. It's better if the ORM doesn't depend on such low-level implementation details at all and uses the schema manager for managing schema.